### PR TITLE
AOM-148:  Fix UI issues.

### PR DESCRIPTION
**Pull request by justmesam**
## JIRA TICKET NAME:
[AOM-148:  Fix UI issues.](https://issues.openmrs.org/browse/AOM-148)
### SUMMARY:
Update the wording of the page where the user is not allowed to see the addon manager.
Also adds consistency in using `Add On` rather than `Add-On`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 <img src="https://cloud.githubusercontent.com/assets/668093/12567089/0ac42774-c372-11e5-97eb-00baf0fccc37.jpg" alt="OpenMRS"/>
 
-# OpenMRS Add-Ons Manager
+# OpenMRS Add Ons Manager
```
Comment (aenkya) [inline README.md:7]: Is it `Add Ons` or `Add On`

Comment (justmesam) [reply README.md:7]: its `Add Ons`

File: tests/components/Addon.test.js [hunk 2]
```diff
@@ -59,6 +60,11 @@ describe('<Addon />', () => {
     expect(component.hasClass('addon')).to.equal(true);
   });
 
+  it('should have addon back button', () => {
+    const backButton = "Back to All Add Ons"
```
Comment (dkayiwa) [inline tests/components/Addon.test.js]: Can you change from All to all?

Comment (justmesam) [reply tests/components/Addon.test.js]: done 👍

**Pull request merged**